packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -220,6 +220,11 @@ export const Table = memo((props: Props) => {
 
       prepareRow(row);
 
+      const isFieldPopulated = (f: Field) => f.values.length > 0;
+      const isDataFramePopulated = (df: DataFrame) => df.fields.some((f: Field) => isFieldPopulated(f));
+      const isRowSubDataPopulated = (dfs: DataFrame[]) => dfs?.some((df: DataFrame) => isDataFramePopulated(df));
```

Comment:
Good solution for this. Just a very small nit; since we updated the nomenclature to "nestedDataField", should we do the same for this function name? `isRowSubDataPopulated` -> `hasNestedDataFieldValues` or something along that line?

Reply:
Oh yes, I completely agree that makes sense. I've just pushed through that exact suggestion, but also thoughts on jthe name `isNestedDataFieldPopulated` to stay consistent with the previous lines? Or changing each of those lines to `hasFieldValues` and `hasDataFrameValues` respectively instead?

(if we even think that level of consistency is important - we could just as easily leave it as is)

Reply:
Honestly, whatever makes sense to you and your future self. Something I wish I learned earlier in the journey is that every line of code I write, isn't written for me. It's written for the future engineer who needs to come in, see the code, understand it, and make changes - which, of course, **_could_** very well be you. The interpreter/compiler couldn't care less what you name something, but your future self might. So, whatever makes sense for you and your future self is the right call. :)